scripts/lib/install

```diff
@@ -99,17 +99,31 @@ export LANG="en_US.UTF-8"
 export LANGUAGE="en_US.UTF-8"
 
 # Check for a supported OS release.
-apt-get install -y lsb-release
-os_info="$(lsb_release --short --id --release --codename)"
-{ read -r os_id; read -r os_release; read -r os_codename; } <<< "$os_info"
+# $os_release and $os_codename are used only when $os_id is Ubuntu, to check
+# whether the Ubuntu Universe repository is enabled.
+if [ -f /etc/debian_version ]; then
+    apt-get install -y lsb-release
+    os_info="$(lsb_release --short --release --codename)"
+    { read -r os_release; read -r os_codename; } <<< "$os_info"
+else
+    export os_release="none"
+    export os_codename="none"
+fi
```

Comment:
I’m trying to get you to remove all uses of `lsb-release` entirely, like we did for the development provisioner, so that we don’t have multiple sources of OS information that we don’t need. Maybe I’m not being explicit enough. What I’m proposing is
1. delete this block,
2. read `VERSION_CODENAME` from `/etc/os-release` as `$os_version_codename` (*unconditionally*—it’s fine if we get an empty string on CentOS, because we only use it on Ubuntu),
3. change all uses of `$os_release` to `$os_version_id` and all uses of `$os_codename` to `$os_version_codename`.

Reply:
Fixed.